crm_phonecall_planner/tests/test_planner.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from datetime import datetime, timedelta
+
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tools import float_compare
+
+from odoo.addons.base.tests.common import BaseCommon
+
+
+class PlannerCase(BaseCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.mondays = cls.env["resource.calendar"].create(
+            {
+                "name": "mondays",
+                "attendance_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "name": "Monday morning",
+                            "dayofweek": "0",
+                            "hour_from": 9,
+                            "hour_to": 12,
+                        },
+                    ),
+                    (
+                        0,
+                        0,
+                        {
+                            "name": "Monday evening",
+                            "dayofweek": "0",
+                            "hour_from": 15,
+                            "hour_to": 20,
+                        },
+                    ),
+                ],
```

Comment:
Could you use the [Command](https://github.com/odoo/odoo/blob/7dcdb774e61e6d520d49dc402c4fad6dcf527ac1/odoo/fields.py#L4227-L4341
) class? It’s more readable.

```suggestion
                "attendance_ids": [
                    Command.create(
                        {
                            "name": "Monday morning",
                            "dayofweek": "0",
                            "hour_from": 9,
                            "hour_to": 12,
                        },
                    ),
                   Command.create(
                        {
                            "name": "Monday evening",
                            "dayofweek": "0",
                            "hour_from": 15,
                            "hour_to": 20,
                        },
                    ),
                ],
```